nautobot/docs/additional-features/jobs.md

```diff
@@ -553,12 +553,9 @@ Provision of user input (`data` values) via the CLI is not supported at this tim
 
 Jobs are Python code and can be tested as such, usually via [Django unit-test features](https://docs.djangoproject.com/en/stable/topics/testing/). That said, there are a few useful tricks specific to testing Jobs.
 
-While individual methods within your Job can and should be tested in isolation, you'll likely also want to test the entire execution of the Job. The simplest way to do so is via calling the `nautobot.extras.jobs.run_job()` method, as this is the same function used to execute a Job via Nautobot's Celery worker process. As part of your test, you can call `run_job` directly rather than enqueuing it for the worker to handle; this is simpler and generally quite sufficient for testing.
+While individual methods within your Job can and should be tested in isolation, you'll likely also want to test the entire execution of the Job. The simplest way to do so is via calling the `nautobot.utilities.testing.run_job_for_testing()` method, as this is the same function used to execute a Job via Nautobot's Celery worker process. As part of your test, you can call `run_job_for_testing()` directly rather than enqueuing it for the worker to handle; this is simpler and generally quite sufficient for testing.
 
-Because of the way `run_job()` works, which is somewhat complex behind the scenes, there are a few things you'll need to do in your test code:
-
-1. Inherit from `django.test.TransactionTestCase` instead of `django.test.TestCase`. (Refer to the [Django documentation](https://docs.djangoproject.com/en/stable/topics/testing/tools/#provided-test-case-classes) if you're interested in the differences between these classes.)
-2. Set your test class's `databases` attribute to `("default", "job_logs")`. (`job_logs` is a proxy connection to the same (`default`) database that's used exclusively for Job logging.)
+Because of the way `run_job_for_testing` and more specifically `run_job()` (which `run_job_for_testing` wraps around) works, which is somewhat complex behind the scenes, you need to inherit from `nautobot.utilities.testing.TransactionTestCase` instead of `django.test.TestCase` (Refer to the [Django documentation](https://docs.djangoproject.com/en/stable/topics/testing/tools/#provided-test-case-classes) if you're interested in the differences between these classes - `TransactionTestCase` from Nautobot is a small wrapper around Django's `TransactionTestCase`.)
 
 !!! tip
     Additionally, *if your test case needs to be backwards-compatible with test execution against Nautobot 1.2.x*, you should add the following block to your test module:
```

Comment:
One downside of this PR is that the test approach it provides is no longer backwards-compatible to Nautobot 1.2.x since that doesn't have `run_job_for_testing` or the updated `TransactionTestCase`. So I guess maybe removing this tip and revising the below example would be appropriate?

Reply:
What do you think about having
- a "pre 1.2" section, potentially with a hint to `run_job_for_testing` + `TransactionTestCase` from 1.3 for copy-pastin
- a "post 1.2" section which is similar to this section?

Reply:
Feels clunky, but maybe our best option for now. Thanks!

Reply:
Gave it a first try 👍🏻